src/main/java/edition/academy/seventh/serivce/BookService.java

```diff
@@ -2,28 +2,38 @@
 
 import edition.academy.seventh.database.model.Book;
 import edition.academy.seventh.repository.BookRepository;
+import java.util.List;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
-
-import java.util.List;
-
-
 /**
  * Service that provides API responsible for adding books into database.
```

Comment:
Poprawić: "Service that provides API responsible for adding books **to** database."
albo "Service that provides API responsible for **inserting** books into database."